runtime/oti/jvminit.h

```diff
@@ -521,6 +524,8 @@ enum INIT_STAGE {
 #define VMOPT_XXIDLETUNINGCOMPACTONIDLEENABLE "-XX:+IdleTuningCompactOnIdle"
 #define VMOPT_XXIDLETUNINGIGNOREUNRECOGNIZEDOPTIONSDISABLE "-XX:-IdleTuningIgnoreUnrecognizedOptions"
 #define VMOPT_XXIDLETUNINGIGNOREUNRECOGNIZEDOPTIONSENABLE "-XX:+IdleTuningIgnoreUnrecognizedOptions"
+#define VMOPT_XCONCURRENTBACKGROUND "-Xconcurrentbackground"
+#define VMOPT_XGCTHREADS "-Xgcthreads"
```

Comment:
Looking to mapping implemented already... Looks like prefix should be `MAPOPT_` instead of `VMOPT_` for this options as well

Reply:
I am taking this comment back. Your implementation is ok

Reply:
This definitions can be used in `mmparse.cpp` to replace hardcoded values. `jvminit.h` is included there already so it is trivial

Reply:
Done see 021929c0faeae4dd5f821bd473134db47df01fda